netlify.toml

```diff
@@ -8,11 +8,11 @@
     X-XSS-Protection = "1; mode=block"
     X-Content-Type-Options = "nosniff"
     Content-Security-Policy = '''
-      font-src 'self' https://*.cloudfront.net data:;
+      font-src 'self' https://*.netlify.app https://*.cloudfront.net data:;
       img-src 'self' data: https:;
-      script-src 'self' 'unsafe-inline' https://*.cloudfront.net *.google-analytics.com *.googletagmanager.com;
-      style-src 'self' 'unsafe-inline' https://*.cloudfront.net;
-      frame-src *.arcana.network *.transak.com *.ramp.network;
+      script-src 'self' 'unsafe-inline' https://*.netlify.app https://*.cloudfront.net *.google-analytics.com *.googletagmanager.com;
+      style-src 'self' 'unsafe-inline' https://*.netlify.app https://*.cloudfront.net;
+      frame-src *.arcana.network *.netlify.app *.transak.com *.ramp.network;
```

Comment:
Can you remove `https://*.netlify.app` if it's not required?

Reply:
[done](https://github.com/arcana-network/wallet-ui/pull/466/commits/8dcf923f3e831e4bacae4d1392d1c9233711408a)